docs/reference/module-options.md

```diff
@@ -372,3 +372,442 @@ module.exports = {
 }
 ```
 
+## Options for the core page module
+
+Option settings in this section apply to the core page module (`@apostrophecms/page`).
+
+| Option | Value type | Description |
+|---------|---------|---------|
+| [`builders`](#builders) | Object | Set query builder values to be used when pages are served. |
+| [`home`](#home) | Boolean/Object | Change how the home page is added to `req.data` when pages are served. |
+| [`minimumPark`](#minimumpark) | Array | Override home page default properties when created. |
+| [`park`](#park) | Array | Set pages to be created on site start with configuration. |
+| [`publicApiProjection`](#publicapiprojection-for-pages) | Object | Set query builder values to be used when pages are served. |
+| [`quickCreate`](#quickcreate-for-pages) | Boolean | Set to `false` to remove pages from the quick create menu. |
+| [`types`](#types) | Array | Set the page types available for new pages. |
+
+### `builders`
+
+<!-- TODO: Update link to a more detailed explanation of builders when
+available. -->
+The `builders` option can be used to apply any existing [query builders](/guide/major-changes.md#queries) when a page is served by its URL. This effects the data available on `req.data.page` (`data.page` in templates). This can be used to get more or less included on that page object depending on the need.
```

Comment:
"This can be used to get more or less included on that page object depending on the need.". Can we rephrase "this can be used to get more or less....". "This can be used to control what is included in the page object"

Reply:
From Ben: "Set pages to be created on site start with configuration." Would this be site creation?"

Bea rightly points out that we can't give every detail in these quick summary tables and we provide more specifics later, but the specifics are:

Parked page settings are applied and, if necessary, re-applied when the process starts up. Also, Apostrophe's UI aims to prevent changes to any parked settings (TODO in A3). If you want to set defaults for certain properties but not enforce them again on every startup, use the `_defaults` subproperty of a parked page and specify defaults for individual fields as subproperties of `_defaults`.